webui/react/src/pages/F_ExpList/F_ExperimentList.settings.ts

```diff
@@ -1,13 +1,16 @@
 import * as t from 'io-ts';
+import { literal, union } from 'io-ts';
 
 import { INIT_FORMSET } from 'components/FilterForm/components/FilterFormStore';
 import { SettingsConfig } from 'hooks/useSettings';
 
 import { defaultExperimentColumns } from './glide-table/columns';
 
+export type ExpListView = 'scroll' | 'paged';
 export interface F_ExperimentListSettings {
   columns: string[];
   filterset: string; // save FilterFormSet as string
+  pageLimit: number;
```

Comment:
nit: previously we've called this same variable `tableLimit`. `pageLimit` is probably more accurate, but I just wanted to bring it up